ONSdigital/census-rm-docker-dev
# Feature/replace pubsub

# Motivation and Context
We are replacing the pubsub dockerized service, adding a queue to rabbitmq and altering some service parameters.

# What has changed
- the docker compose yaml has changed to support the new service and parameters
- the rabbitmq definitions file now has the new queue within it

# How to test?
- run the acceptance tests locally
- ensure the CI acceptance tests run correctly
